3_visualize/src/plot_stripswarm.R

```diff
@@ -24,4 +24,51 @@ event_swarm_plot_compressed_vertical <- function(swarm_data){
                  expand = c(0,0),
                  position = 'right')
   
-}
\ No newline at end of file
+}
+
+
+
+event_violin_vertical <- function(drought_data, major_drought_periods, color_scheme){
+  # focal CASC
+  focal_CASC <- unique(drought_data$CASC)
+  
+  # plot violin vertically
+
+  major_drought_periods |> 
+    ggplot(aes(y = start,
+               x = xfill)) +
+    # drought events shaded
+    geom_ribbon(aes(ymin = start, ymax = end,
+                    group = name),
+                fill = color_scheme$drought_period_shading, alpha = 0.7) +
+    ggdist::geom_dots(data = drought_data,
+                      aes(y = date, x = threshold),
+                      color = color_scheme$drought_event_highlight,
+                      fill = NA, side = "both") +
+    theme_nothing() + 
+      theme(axis.text.x = element_blank(),
+            axis.text.y = element_text(size = 6, 
+                                       color = color_scheme$annotation_grey, 
+                                       angle = 180),
+            panel.grid.major.y = element_line(color = color_scheme$annotation_grey, 
+                                              linewidth = 0.2),
+            plot.title = element_text(size = 8, color = color_scheme$annotation_grey, 
+                                      angle = 180)) +
+      scale_y_date(breaks = scales::date_breaks(width = '5 years'),
+                   labels = scales::date_format('%Y'), 
+                   limits = c(as.Date("1920-01-01"), 
```

Comment:
SC: For this and the radial plot, I think it would be better to define these dates programmatically, instead of hard-coding them. Like we did in `event_swarm_plot_compressed_vertical()`

Reply:
I don't know, this looks a little awkward, because it doesn't have the year 1920 on it now: 

![vertical_violin_jd7d_2pct_Northwest](https://github.com/DOI-USGS/drought-timeline/assets/19958841/ece66892-72c6-4d36-86a4-ad27654abc07)

Reply:
Yeah, that's not ideal. Maybe leave as it was then, w/ a note that start dates vary across the CASCs, so  you're setting it manually